{"name": "修改密码成功", "status": "broken", "statusDetails": {"message": "AttributeError: 'tuple' object has no attribute 'url'", "trace": "@pytest.fixture(scope='session')\n    def token():\n        url='http://qa.yansl.com:8084/login'\n        req={\n             \"pwd\": \"123456zjc\",\n             \"userName\": \"123zjc\"\n            }\n>       resp = request_tool.post_json(url, json=req)\n\ntest_case\\conftest.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ('http://qa.yansl.com:8084/login',)\nkwargs = {'json': {'pwd': '123456zjc', 'userName': '123zjc'}}, r = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError"}, "start": 1602684365815, "stop": 1602684365815, "uuid": "4b3b6d2f-fa57-4b2a-ae91-2c2d19bb281f", "historyId": "36fee43a9f2b3f263132e9d44d7dc434", "fullName": "test_case.User.test_changepwd#test_changepwd", "labels": [{"name": "parentSuite", "value": "test_case.User"}, {"name": "suite", "value": "test_changepwd"}, {"name": "subSuite"}, {"name": "host", "value": "DESKTOP-679Q8H2"}, {"name": "thread", "value": "9868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.User.test_changepwd"}]}